src/Components/Navbar/Navbar.js

```diff
@@ -2,10 +2,12 @@ import "./Navbar.css";
 import { Link } from "react-router-dom";
 import { useAuth } from "../../context/auth-context";
 import { useWishlist } from "../../context/wishlist-context";
+import { useCart } from "../../context/cart-context";
```

Comment:
all custom hooks can go into the single file and from that you can import

Reply:
ok make sense I will do the barrel imports with this one.